heron/spi/src/java/com/twitter/heron/spi/utils/ShellUtils.java

```diff
@@ -111,11 +112,24 @@ public static Process runASyncProcess(
       boolean verbose, String[] command, File workingDirectory) {
     // Log the command for debugging
     LOG.log(Level.FINE, "$> {0}", Arrays.toString(command));
+    // Log the working directory, this will make people fast locate the
+    // directory to find the log files
+    LOG.log(Level.INFO, "working dir: {0}", workingDirectory.getAbsolutePath());
```

Comment:
@ashvina can provide better ideas when this logging should be.

Is there a way to easily adjust the log level.
People could submit topology with verbose flag, and then expect the logging level be Level.ALL (https://github.com/twitter/heron/blob/master/heron/scheduler-core/src/java/com/twitter/heron/scheduler/SubmitterMain.java#L275). 
Admittedly this is not fully done, for instance, not yet done in SchedulerMain or HeronExecutorTask. Pull request for them are welcome.